docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+### Geomap Network layer
+
+_Available in public preview in all editions of Grafana_
+
+<!-- Nathan Marrs -->
+
+You can now display network data in the Geomap visualization by using the new beta Network layer. This layer supports the same data format as the [Node graph visualization]({{< relref "../panels-visualizations/visualizations/node-graph/#data-api" >}}).
+
+To learn more, refer to our [Geomap network layer documentation]({{< relref "../panels-visualizations/visualizations/geomap/#network-layer-beta" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-geomap-network-layer-v2.png" max-width="750px" caption="Geomap Network layer" >}}
+
+### Heatmap visualizations now support data links
+
+_Generally available in all editions of Grafana_
+
+<!-- Nathan Marrs -->
+
+You can now add data links to Heatmap visualizations. This allows you to add links to other dashboards, panels, or external URLs that are relevant to the data in your heatmap. We're pleased to highlight that this feature was a community contribution.
+
+To learn more, refer to both our [Heatmap documentation]({{< relref "../panels-visualizations/visualizations/heatmap/" >}}) and our [Configure data links documentation]({{< relref "../panels-visualizations/configure-data-links/" >}}).
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-heatmap-datalinks.png" max-width="750px" caption="Heatmap datalink support" >}}
+
+## Data sources
+
+### Step editor in Loki
+
+<!-- Ivana Huckova -->
+
+_Generally available in all editions of Grafana_
+
+We've improved the Loki query editor by adding a new **Step** editor field. This editor allows you to specify a value for the `step` parameter in Loki queries. You can use this parameter when making metric queries to Loki or when you want a matrix response from your queries.
+
+By default, the `step` parameter is set to the value of the `$__interval` variable. This variable is calculated based on the time range and the width of the graph (in pixels). If you want to learn more about the Loki `step` parameter, you can visit [the Loki step parameter documentation](/docs/loki/latest/api/#step-versus-interval).
+
+{{< figure src="/media/docs/grafana/data-sources/loki-step-editor.png" max-width="750px" caption="New Loki step editor" >}}
+
+### Copy link to a Loki log line
+
+<!-- Sven Grossmann -->
+
+_Generally available in all editions of Grafana_
+
+A new linking of Loki log lines in Explore allows you to quickly navigate to specific log entries for precise analysis. By clicking the **Copy shortlink** button for a log line, you can generate and copy a [short URL]({{< relref "../developers/http_api/short_url/" >}}) that provides direct access to the exact log entry within an absolute time range. When you open the link, Grafana automatically scrolls to the corresponding log line and highlights it with a blue background, making it easy to identify and focus on the relevant information.
```

Comment:
@svennergr - Is the background blue in light mode because it doesn't appear blue in dark mode? Can we just take out the colour and just say it'll be highlighted? 

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
It should be blue in both modes. E.g. https://ops.grafana-ops.net/goto/v87rFd64R?orgId=1

<img width="519" alt="image" src="https://github.com/grafana/grafana/assets/8092184/f5b64712-72f6-4136-9825-616d77e4a0dc">

Reply:
Nevertheless I am also fine with removing the color from the text.